tools/jenkins-slave-creation-unix/conf-ubuntu-gpu/install.sh

```diff
@@ -59,8 +59,13 @@ echo "Installed htop, java, git and python"
 #Install nvidia drivers
 #Chose the latest nvidia driver supported on Tesla driver for Ubuntu18.04
 #Refer : https://www.nvidia.com/Download/driverResults.aspx/158191/en-us
-sudo apt-get -y install nvidia-driver-435
-sudo apt-get -y install nvidia-utils-435
+wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-ubuntu1804.pin
+sudo mv cuda-ubuntu1804.pin /etc/apt/preferences.d/cuda-repository-pin-600
+wget http://developer.download.nvidia.com/compute/cuda/10.2/Prod/local_installers/cuda-repo-ubuntu1804-10-2-local-10.2.89-440.33.01_1.0-1_amd64.deb
```

Comment:
Why use the local repo instead of the network repo?

Reply:
Acc to https://github.com/NVIDIA/DIGITS/blob/master/docs/InstallCuda.md updates won't be installed if local one is used [also its better when we have poor network which I interpreted as better suited even in poor network? meaning more reliable?

Reply:
Well, you need to download essentially all cuda related packages once to create the local repo. Whereas otherwise you only download the required packages.
This is only run once during AMI creation, so it may be better not to hardcode `10.2.89-440.33.01_1.0-1_amd64` version but rely on the latest version of cuda 10.2 available at time of ami creation?

Reply:
Right. Makes sense. Missed that point. Thanks for bringing it up.

Reply:
Yeah let's not use local repo.

Reply:
Updated code and AMI with network instead of local